modules/calib3d/src/stereosgbm.cpp

```diff
@@ -705,7 +705,7 @@ static void computeDisparitySGBM( const Mat& img1, const Mat& img2,
                                 L0 = L0 + v_load(Sp + d);
                                 v_store(Sp + d, L0);
 
-                                v_int16x8 mask = _minS > L0;
+                                v_int16x8 mask = v_int16x8::fromMask(_minS > L0);
```

Comment:
Can we keep this code "as is" at least for SIMD128? (make it backward compatible, see opencv_contrib)

SIMD256/SIMD512/wide universal intrinsics - new "mask" type.

Reply:
I think we can't that will break dispatch headers and there's no need to workaround it since ::fromMask only reinterpret mask types for SIMD/128&256.
So we're not expecting any performance cost and of course I will provide performance testing results to make sure that everything is okay.

BTW I moved AVX512 compatibility commits into separate PR #14331 and this PR should be merge after #14232 and backport of #14331.